docs/src/modules/components/ApiPage.js

```diff
@@ -1,123 +1,15 @@
 /* eslint-disable react/no-danger */
 import * as React from 'react';
 import PropTypes from 'prop-types';
-import clsx from 'clsx';
 import { exactProp } from '@mui/utils';
-import { alpha, styled } from '@mui/material/styles';
-import Alert from '@mui/material/Alert';
 import Typography from '@mui/material/Typography';
 import { useTranslate, useUserLanguage } from 'docs/src/modules/utils/i18n';
+import PropsTable from 'docs/src/modules/components/PropretiesTable';
```

Comment:
It would be better if it matches

```suggestion
import PropretiesTable from 'docs/src/modules/components/PropretiesTable';
```

Reply:
Yep, will update 👍